ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -237,4 +237,33 @@ Public Class ucrFormDaily2
         Next
     End Sub
 
+    Public Sub checkTotal()
+        'Check total if required
+        ' Am not sure how to get this value yet
+        'If totalRequired = 1 Then
+
+        Dim elemTotal As Integer = 0
+        Dim expectedTotal As Integer
+        Dim ctr As Control
+        Dim ctrVFP As New ucrValueFlagPeriod
+
+        expectedTotal = ucrInputTotal.GetValue
+
+        For Each ctr In Me.Controls
+                If TypeOf ctr Is ucrValueFlagPeriod Then
+                    ctrVFP = ctr
+                    elemTotal = elemTotal + ctrVFP.ucrValue.GetValue
+                End If
+            Next
+
+        If elemTotal <> expectedTotal Then
+            MessageBox.Show("Value in [Total] textbox is different from that calculated by computer!", caption:="Error in total")
+            ucrInputTotal.txtBox.Focus()
+            ucrInputTotal.txtBox.BackColor = Color.Cyan
+        End If
+    End Sub
+
+    Private Sub ucrInputTotal_Leave(sender As Object, e As EventArgs) Handles ucrInputTotal.Leave
```

Comment:
I put this on the the leave event of the total textbox. I am not sure we want it here or just run this when people click on save which was what samuel had done

Reply:
I like this here. The question is how this check is then registered? A colour? A Message box? Can we save it if they don't match?

Reply:
I am not quite sure if we need to save it if they dont match. I need to look further into how that was done. 
However, that was not a critical warning and hence i think it was saved if people wanted to continue after throwing a warning message.